src/pages/login/CoreConnect.tsx

```diff
@@ -65,8 +69,47 @@ const CoreConnect = () => {
       });
   };
 
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('http://132.145.101.179:42069/');
+        if (!response.ok) {
+          setShowPopup(true);
+        }
+        else {
+        }
+      }
+      catch (error) {
+        setShowPopup(true);
+        console.log("fail");
+      }
+    };
+
+    fetchData();
+  }, []);
+
   return (
     <div className="flex w-[768px] max-w-full flex-col items-stretch justify-center gap-12 rounded-2xl px-12 py-14 @container">
+      {showPopup && (
+        <div>
+          <ConfirmDialog
+            isOpen={showPopup}
+            onClose={() => setShowPopup(false)}
+            title="HTTP Error"
+            type="info"
+          >
+            Missing user permissions. Enable mixed content settings to load HTTP content over HTTPS. {' '}
+            <a
+              href="https://github.com/Lodestone-Team/lodestone/wiki/Known-Issues#network-errors"
+              target="_blank"
+              rel="noreferrer"
+              className="text-blue-200 underline hover:text-blue-300"
+            >
+            Learn more.
+            </a>          
+          </ConfirmDialog>          
+        </div>
+      )}
       <div className="flex flex-col items-start">
         <img src="/logo.svg" alt="logo" className="h-8" />
         <h1 className="font-title text-h1 font-bold tracking-medium text-gray-300">
```

Comment:
I think we can remove the "You may need to adjust your network and browser settings" warning that's always there, if we are sure the user can access http content